interface-definitions/system-sflow.xml.in

```diff
@@ -0,0 +1,112 @@
+              </completionHelp>
+              <valueHelp>
+                <format>ipv4</format>
+                <description>sFlow IPv4 agent address</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6</format>
+                <description>sFlow IPv6 agent address</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ipv4-address"/>
+                <validator name="ipv6-address"/>
+                <validator name="ipv6-link-local"/>
+              </constraint>
+            </properties>
+          </leafNode>
+          <leafNode name="agent-interface">
+            <properties>
+              <help>IP address associated with this interface</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces</script>
+              </completionHelp>
+              <valueHelp>
+                <format>txt</format>
+                <description>Interface name</description>
+              </valueHelp>
+              <constraint>
+                #include <include/constraint/interface-name.xml.in>
+              </constraint>
+            </properties>
+          </leafNode>
+          #include <include/generic-interface-multi.xml.i>
+          <leafNode name="polling">
+            <properties>
+              <help>Schedule counter-polling in seconds</help>
+              <valueHelp>
+                <format>u32</format>
+                <description>Polling rate in seconds</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-600"/>
+              </constraint>
+            </properties>
+            <defaultValue>30</defaultValue>
+          </leafNode>
+          <leafNode name="sampling-rate">
+            <properties>
+              <help>sFlow sampling-rate</help>
+              <valueHelp>
+                <format>u32</format>
+                <description>Sampling rate (1 in N packets)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-65535"/>
+              </constraint>
+            </properties>
+            <defaultValue>1000</defaultValue>
+          </leafNode>
+          <tagNode name="server">
+            <properties>
+              <help>sFlow destination server</help>
+              <valueHelp>
+                <format>ipv4</format>
+                <description>IPv4 server to export sFlow</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6</format>
+                <description>IPv6 server to export sFlow</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ipv4-address"/>
+                <validator name="ipv6-address"/>
+              </constraint>
+            </properties>
+            <children>
+              <leafNode name="port">
```

Comment:
Is it definitely not possibly to bind as root on privileged ports (<1024)? If it's possible - please use the generic port-number XML building block.

Reply:
It depends on whether the remote site (collector) can listen to ports <1024 for sFlow 
We don't bind those ports locally
Will do.

Reply:
Done!